### PR TITLE
Add Fry/DEA App

### PR DESCRIPTION
**Pull request by nathanielbuck**
## Description
Add a new form application for Fry/DEA.

Sibling PR: https://github.com/department-of-veterans-affairs/content-build/pull/1159

### CODE REVIEW

**Review by theodur — COMMENTED** [review]
Please feel free to add this app to the [allow-list](https://github.com/department-of-veterans-affairs/vets-website/blob/main/config/changed-apps-build.json), so it can take advantage of faster CI times.

Also, it looks like there's  a failed unit test related to your app.

**Review by smeneni — APPROVED** [review]

**Review by asg5704 — APPROVED** [review]
No impact on my files. LGTM

**Review by it-harrison — APPROVED** [review]

File: src/platform/forms/save-in-progress/SaveInProgressIntro.jsx [hunk 1]
```diff
@@ -160,7 +160,7 @@ class SaveInProgressIntro extends React.Component {
             </div>
           );
         }
-      } else if (prefillAvailable && !verifiedPrefillAlert) {
+      } else if (!prefillAvailable && !verifiedPrefillAlert) {
```
Comment (bkjohnson) [inline src/platform/forms/save-in-progress/SaveInProgressIntro.jsx:163]: Why are things changing here?

**Review by micahchiang — CHANGES_REQUESTED** [review]
This pull request is big, even for one that adds a new application. Was main properly merged into this branch before submitting this PR? Looks like a ton of files that have nothing to do with the DEA app have been included.

**Comment by nathanielbuck** [comment]
I'm not exactly sure what happened after my initial commit (I've been away for the past week). I think @smeneni may have pushed a bad merge/rebase. I'm going to close this and open a fresh PR.

File: src/platform/forms/save-in-progress/SaveInProgressIntro.jsx (hunk 1 above)
Comment (nathanielbuck) [reply src/platform/forms/save-in-progress/SaveInProgressIntro.jsx:163]: @smeneni, this isn't the correct place to make these changes. In general, we should not be making changes to platform code.

**Pull request closed without merging**